ParkingLot/src/main/java/Application.java

```diff
@@ -1,61 +0,0 @@
-import dao.TicketsDao;
-import factories.CostComputationFactory;
-import factories.ParkingSpotManagerFactory;
-import models.ParkingSpots.FourWheelerParkingSpot;
-import models.ParkingSpots.ParkingSpot;
-import models.ParkingSpots.TwoWheelerParkingSpot;
-import models.Ticket;
-import models.Vehicle;
-import models.VehicleType;
-import services.EntranceGate;
-import services.ExitGate;
-
-
-import java.util.ArrayList;
-import java.util.List;
-
-public class Application {
-
-    public static void main(String[] args) throws Exception {
-        TicketsDao ticketsDao = new TicketsDao();
-        CostComputationFactory costComputationFactory = new CostComputationFactory();
-        List<ParkingSpot> twoWheelerParkingSpotList = new ArrayList<>();
-        List<ParkingSpot> fourWheelerParkingSpotList = new ArrayList<>();
-        List<Ticket> ticketList = new ArrayList<>();
```

Comment:
ticketList(map) is already there in InMemoryDataStore?

Reply:
Yup, you are right. The intention to maintain this list was mainly logging and maintaining the customer copy of the ticket. I should have validated this ticket from my inMemoryDataStore.